data/grc/Erbse_1961_Aischylos-Persai,HeptaepiThebas,Hiketides,Prometheusdesmotes,Agamemnon,Choephoroi,Eumenides/metadata.txt

```diff
@@ -0,0 +1,82 @@
+publicationType : "chapter"
+publicationTitle : "Überlieferungsgeschichte der griechischen klassischen und hellenistischen Literatur"
+publicationDate : "1961"
+publicationPlace : "Zürich"
+publicationSeries : "Geschichte der Textüberlieferung der antiken und mittelalterlichen Literatur"
+publicationNum: "1"
+publicationStemmaNum: ""
+publicationAuthors : "Erbse, Hartmut"
+publicationPage : "274"
+publicationLink : "http://opac.regesta-imperii.de/lang_de/anzeige.php?sammelwerk=Geschichte+der+Text%C3%BCberlieferung+der+antiken+und+mittelalterlichen+Literatur"
+workTitle : "Persai"
+workViaf : "183898734"
+workTitle : "Hepta epi Thebas"
+workViaf : "189401003"
+workTitle : "Hiketides"
+workViaf : "278730739"
+workTitle : "Prometheus desmotes"
+workViaf : "315652031"
+workTitle : "Agamemnon"
+workViaf : "213089523"
+workTitle : "Choephoroi"
+workViaf : "310935937"
+workTitle : "Eumenides"
+workViaf : "188652016"
+workOrigDate : "6th/5th century BCE"
```

Comment:
I would advocate to have a single system for dates (including ranges), of the form
`\d+(\-\d+)?`

and then something like:
> 600-401

Reply:
Agress